examples/src/main/kotlin/io/grpc/examples/routeguide/RouteGuideServer.kt

```diff
@@ -127,8 +125,8 @@ class RouteGuideServer private constructor(
       }.build()
     }
 
-    override fun routeChat(requests: Flow<RouteNote>): Flow<RouteNote> {
-      return flow {
+    override fun routeChat(requests: Flow<RouteNote>): Flow<RouteNote> =
+      flow {
         // could use transform, but it's currently experimental
```

Comment:
@lowasser - do you find that this comment adds value for the tutorial readers? (If not, shall we drop it?)

Reply:
(If you'd like to keep the comment in the code, I can drop it from the tutorial code excerpt only.)

Reply:
Let's drop it.

Reply:
Done.